packages/terra-header/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "terra-header",
+  "main": "lib/Header.js",
+  "version": "1.0.0",
+  "description": "The terra-header component is a styled layout of a header bar.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-header",
+    "Header",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-button": "^3.17.0",
+    "terra-doc-template": "^2.12.0",
+    "terra-icon": "^3.13.0",
```

Comment:
It looks like this can be removed, I didn't see it used in this PR.

Reply:
Fixed Here https://github.com/cerner/terra-core/pull/2545/commits/c5311e2840aab180a76df4165fbbb158ab911ba3